src/pages/docs.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+	<div class="flex mx-auto w-full max-w-7xl">
+		<Sidebar v-if="docs" />
+		<router-view v-if="docs" :key="$route.path" />
+		<Spinner v-else />
+	</div>
+</template>
+
+<script setup lang="ts">
+import { reactive, computed } from 'vue';
+import { useRouter, useRoute } from 'vue-router';
+import { useStore } from '~/store';
+
+import MainSource from '~/data/MainSource';
+import CollectionSource from '~/data/CollectionSource';
+import CommandoSource from '~/data/CommandoSource';
+import RPCSource from '~/data/RPCSource';
+import DocsSource from '~/data/DocsSource';
+
+import Sidebar from '~/components/Sidebar.vue';
+import Spinner from '~/components/Spinner.vue';
+
+const router = useRouter();
+const route = useRoute();
+const store = useStore();
+
+const sources = reactive({
+	[MainSource.id]: MainSource,
+	[CollectionSource.id]: CollectionSource,
+	[CommandoSource.id]: CommandoSource,
+	[RPCSource.id]: RPCSource,
+});
+
+const source = computed(() => store.state.source);
+const tag = computed(() => store.state.tag);
+const docs = computed(() => store.state.docs);
+
+const fetchDocs = async (inputSource: DocsSource, inputTag: string) => {
+	const documentation = await inputSource.fetchDocs(inputTag);
+
+	// Sort everything
+	documentation.classes.sort((a: { name: string }, b: { name: string }) => a.name.localeCompare(b.name));
+	documentation.typedefs.sort((a: { name: string }, b: { name: string }) => a.name.localeCompare(b.name));
+	for (const c of documentation.classes) {
+		if (c.props) {
+			c.props.sort((a: { name: string }, b: { name: string }) => a.name.localeCompare(b.name));
+		}
+		if (c.methods) {
+			c.methods.sort((a: { name: string }, b: { name: string }) => a.name.localeCompare(b.name));
+		}
+		if (c.events) {
+			c.events.sort((a: { name: string }, b: { name: string }) => a.name.localeCompare(b.name));
+		}
+	}
+
+	// Built-in type links
+	documentation.links = {
+		string: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String',
+		number: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number',
```

Comment:
```suggestion
		number: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number',
		bigint:  'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt',
```
Since we have bigints now....